<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Magazzino</class>
 <widget class="QDialog" name="Magazzino">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>422</width>
    <height>300</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="5">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Crea magazzino</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string/>
         </property>
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="1" column="1">
           <widget class="QLineEdit" name="lineMagazzino">
            <property name="placeholderText">
             <string>Inserisci nome magazzino</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Crea nuovo magazzino</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>109</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>ID</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="id">
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Lista magazzino</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="QTableView" name="tableView"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="nuovo">
     <property name="text">
      <string>Nuovo</string>
     </property>
     <property name="icon">
      <iconset resource="../fabaria.qrc">
       <normaloff>:/images/dialog-ok-apply.png</normaloff>:/images/dialog-ok-apply.png</iconset>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QPushButton" name="save_magazzino">
     <property name="text">
      <string>Salva</string>
     </property>
     <property name="icon">
      <iconset resource="../fabaria.qrc">
       <normaloff>:/images/document-save.png</normaloff>:/images/document-save.png</iconset>
     </property>
    </widget>
   </item>
   <item row="1" column="4">
    <widget class="QPushButton" name="predefinito">
     <property name="text">
      <string>Rendi predefinito</string>
     </property>
     <property name="icon">
      <iconset resource="../fabaria.qrc">
       <normaloff>:/images/predefinito.png</normaloff>:/images/predefinito.png</iconset>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="3">
    <widget class="QPushButton" name="elimina">
     <property name="text">
      <string>Elimina</string>
     </property>
     <property name="icon">
      <iconset resource="../fabaria.qrc">
       <normaloff>:/images/edit-delete.png</normaloff>:/images/edit-delete.png</iconset>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>id</tabstop>
  <tabstop>lineMagazzino</tabstop>
  <tabstop>nuovo</tabstop>
  <tabstop>save_magazzino</tabstop>
  <tabstop>elimina</tabstop>
  <tabstop>predefinito</tabstop>
  <tabstop>tabWidget</tabstop>
  <tabstop>tableView</tabstop>
 </tabstops>
 <resources>
  <include location="../fabaria.qrc"/>
 </resources>
 <connections/>
</ui>
